fluidefi/fluidefi-caspernet-aggregator-tools
# Block hours saver

**Summary of change:**
- Added a new micro-service, block hours saver, which accepts block hour fields as an argument and returns the block height and block hour fields. 

**List of tasks:** 

- [x] create fixture.

**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/block-hours-saver`
```bash
git checkout feature/block-hours-saver
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm block-hour-saver test has passed.
- Run example.
